settings.py

```diff
@@ -247,6 +247,7 @@
 
 CRISPY_TEMPLATE_PACK = 'bootstrap3to5'
 CRISPY_ALLOWED_TEMPLATE_PACKS = "bootstrap3to5"
+CRISPY_FAIL_SILENTLY = not DEBUG
```

Comment:
I think this will always be false because `DEBUG` will always be true here. Did you intend to put this line below where `localsettings` is imported?

Reply:
Ah, localsettings 😢 

https://github.com/dimagi/commcare-hq/pull/34952